pages/resources/user.mdx

```diff
@@ -767,6 +791,72 @@ Verifies a verification code sent to the current user's email address to initiat
 | ----- | ------ | -------------------------------------------------------------------------------------------------------- |
 | token | string | The email token; can be used with [Modify Current User](#modify-current-user) to change the user's email |
 
+<RouteHeader method="GET" url="/users/@me/pomelo-suggestions">
+  Get Pomelo Suggestions
+</RouteHeader>
+
+Returns a list of suggested unique username strings based on the current user's username.
+
+<Alert type="warn">
+
+This endpoint is used during the pomelo migration flow. The user must be in the rollout to use this endpoint. See the [section on Discord's new username system](#unique-usernames) for more information.
+
+</Alert>
+
+###### Example Response
+
+```json
+["gnarp", "gnap"]
+```
+
+<RouteHeader method="POST" url="/users/@me/pomelo-attempt">
+  Get Pomelo Eligibility
+</RouteHeader>
+
+Checks whether a unique username is available for the user to claim.
+
+<Alert type="warn">
+
+This endpoint is used during the pomelo migration flow. The user must be in the rollout to use this endpoint. See the [section on Discord's new username system](#unique-usernames) for more information.
+
+</Alert>
+
+###### JSON Params
+
+| Field    | Type   | Description           |
+| -------- | ------ | --------------------- |
+| username | string | The username to check |
+
+###### Response Body
+
+| Field | Type    | Description                   |
+| ----- | ------- | ----------------------------- |
+| taken | boolean | Whether the username is taken |
+
+###### Example Response
+
+```json
+{ "taken": true }
+```
+
+<RouteHeader method="POST" url="/users/@me/pomelo">
+  Pomelo User
```

Comment:
```suggestion
  Create Pomelo Migration
```

since it creates a migration i.e. adds the badge, converts the discriminator + changes username